### ISSUE TITLE
[MODEL] Розробка та тестування моделей на Tensorflow та PyTorch

### ISSUE THREAD

**Issue by bond1q-ua** [carerakjan/goit-telecom-project-group-1#24]
Розробити ти затестувати в Колабі моделі на PyTorch та TensorFlow.

- [Twnsorflow model](https://colab.research.google.com/drive/1E9DQFBcJnjIY3uEgY5lXCDQhVgEM5Tzb?usp=sharing)- (ReLU, Sigmoid)
**Для покращення моделі було додано # Зміна оптимізатора (Adam) і зниження learning rate**
Результати:
Test Accuracy: 0.9287850337297447
Precision: 0.9435483870967742
Recall: 0.9285714285714286
F1 Score: 0.936

- [PyTorch model](https://colab.research.google.com/drive/1f7syCGfH6ysUx63trp0sqBzhOHJIsvyW?usp=sharing)- (ReLU, Sigmoid)
Результати:
Test Accuracy: 0.9282286668057584
Precision: 0.9433795712484238
Recall: 0.927703373015873
F1 Score: 0.9354758034262848
